src/Magento/FunctionalTestingFramework/Util/Script/ScriptUtil.php

```diff
@@ -101,7 +101,7 @@ public function getModuleXmlFilesByScope($modulePaths, $scope)
             if (!realpath($modulePath . $scopePath)) {
                 continue;
             }
-            $finder->files()->followLinks()->in($modulePath . $scopePath)->name("*.xml");
+            $finder->files()->followLinks()->in($modulePath . $scopePath)->name("*.xml")->sortByName();
```

Comment:
Have you checked if this sorting has any side effect on other scripts?

Reply:
no side effects observed, it's only used in deprecated entity test which is covered by verification test